bin/multielasticdump

```diff
@@ -217,7 +218,8 @@ var dump = function () {
       '--output=' + outputData,
       '--scrollTime=' + options.scrollTime,
       '--limit=' + options.limit,
-      '--offset=' + options.offset
+      '--offset=' + options.offset,
+      '--searchBody=' + JSON.stringify(options.searchBody)
```

Comment:
This is actually incorrect. The `searchBody`  shouldn't have been parsed since elasticdump already does this. Should just be passed directly. https://github.com/taskrabbit/elasticsearch-dump/blob/38bdb54f447e84cb00b3a06fc20fe4b817faf783/bin/multielasticdump#L60

Reply:
But it is parsed. Should I remove the line you marked?

Reply:
Parsing inside `multielasticdump` is redundant since it's already been done here :

https://github.com/taskrabbit/elasticsearch-dump/blob/master/bin/elasticdump#L48. Remove the parse logic and pass the raw string to **elasticdump**